pkg/diff/chunk.go

```diff
@@ -85,32 +81,10 @@ func (c *ChunkRange) toString(collation string) (string, []string) {
 		collation = fmt.Sprintf(" COLLATE '%s'", collation)
 	}
 
-	if c.Mode != bucketMode {
-		conditions := make([]string, 0, 2)
-		args := make([]string, 0, 2)
-
-		for _, bound := range c.Bounds {
-			if len(bound.Lower) != 0 {
-				conditions = append(conditions, fmt.Sprintf("`%s`%s %s ?", bound.Column, collation, bound.LowerSymbol))
-				args = append(args, bound.Lower)
-			}
-			if len(bound.Upper) != 0 {
-				conditions = append(conditions, fmt.Sprintf("`%s`%s %s ?", bound.Column, collation, bound.UpperSymbol))
-				args = append(args, bound.Upper)
-			}
-		}
-
-		if len(conditions) == 0 {
-			return "TRUE", nil
-		}
-
-		return strings.Join(conditions, " AND "), args
-	}
-
 	/* for example:
 	there is a bucket in TiDB, and the lowerbound and upperbound are (v1, v3), (v2, v4), and the columns are `a` and `b`,
-	this bucket's data range is (a > v1 or (a == v1 and b >= v2)) and (a < v3 or (a == v3 and a <= v4)),
-	not (a >= v1 and a <= v3 and b >= v2 and b <= v4)
+	this bucket's data range is (a > v1 or (a == v1 and b > v3)) and (a < v2 or (a == v2 and a <= v4)),
```

Comment:
why change it?

Reply:
the old comment is wrong, v1 and v2 is column a's range, v3 and v4 is column b's range. And I change the chunk's range's expression, so need change this example.

Reply:
is the old implementation wrong

Reply:
no, just comment is wrong.